docs/en/query_language/functions/hash_functions.md

```diff
@@ -305,10 +305,42 @@ SELECT murmurHash3_128('example_string') AS MurmurHash3, toTypeName(MurmurHash3)
 └──────────────────┴─────────────────┘
 ```
 
-## xxHash32, xxHash64
+## xxHash32, xxHash64 {#hash_functions-xxhash32}
 
-Calculates xxHash from a string.
-Accepts a String-type argument. Returns UInt64 Or UInt32.
-For more information, see the link: [xxHash](http://cyan4973.github.io/xxHash/)
+Calculates `xxHash` from a string. It is proposed in two flavors, 32 and 64 bits.
```

Comment:
What version of xxHash is in use?

Reply:
By default - it's xxHash v0.6.5 from lz4, unless some custom build is used. 
https://github.com/lz4/lz4/blob/7a4e3b1fac5cd9d4ec7c8d0091329ba107ec2131/lib/xxhash.h#L152
Afaik different xxhash versions are compatible.

By the way, @alexey-milovidov : I know you like different hashing algorithms :) xxHash author had presented a new algorithm that year (still considered experimental) - see https://fastcompression.blogspot.com/2019/03/presenting-xxh3.html

Reply:
Thanks!